svf-llvm/lib/CHGBuilder.cpp

```diff
@@ -96,6 +107,8 @@ void CHGBuilder::buildCHGNodes(const GlobalValue *globalvalue)
         if (!chg->getNode(className))
             createNode(className);
 
+        vtblStruct = maybeGetInnerVtblStruct(vtblStruct);
```

Comment:
I would suggest that you make an option here to use AddressSanVtblStruct. There are few other places where `const ConstantStruct *vtblStruct = SVFUtil::dyn_cast<ConstantStruct>(globalvalue->getOperand(0));`, you may need to make sure we all call AddressSanVtblStruct. Why not wrap up `const ConstantStruct *vtblStruct = getVtblStruct()`; and put everything in `getVtblStruct`, including your method and the assert.